src/libs/swapAndBridge/swapAndBridge.ts

```diff
@@ -74,13 +74,25 @@ export const sortPortfolioTokenList = (accountPortfolioTokenList: TokenResult[])
  * Not all tokens in the portfolio are eligible.
  */
 export const getIsTokenEligibleForSwapAndBridge = (token: TokenResult) => {
+  const getAmount = (t: TokenResult): bigint => {
+    if (typeof t.amountPostSimulation === 'bigint') {
+      // Prevents the token from disappearing from the list if the
+      // amountPostSimulation equals the full token amount
+      if (t.amountPostSimulation === 0n && t.amount > 0n) return t.amount
+
+      return t.amountPostSimulation
+    }
+
+    return t.amount
+  }
```

Comment:
IMO this is confusing, it conflicts with the way the other helper `getTokenAmount` works (gets `amountPostSimulation` or the `amount` of a token).

Either:

1) Maybe move this next to `src/ambire-common/src/libs/portfolio/helpers.ts` and name it getTokenAmountRelevantForSwapAndBridge maybe (to indicate the diff)?

or

2) How about keeping `Number(getTokenAmount(token)) > 0` but also `&& shouldPreventTokenFromDissapearing` where this is something like:

```
const shouldPreventTokenFromDissapearing = typeof token.amountPostSimulation === 'bigint' && token.amountPostSimulation === 0n && t.amount > 0n
```